### DISCUSSION TITLE
Use-case testing of tools and protocols

### DISCUSSION THREAD

**Discussion by samikanza** [MADICES/MADICES-2025#2] [Ideas]
**MADICES 2024 Recap**
Past MADICES workshops underscored the complexities associated with achieving seamless interoperability among diverse RDM platforms. Discussions centered on developing platform-agnostic solutions to facilitate the exchange of research data without being constrained by proprietary APIs. These efforts culminated in the conceptualization of [a new transfer protocol](https://github.com/MADICES/interop-specs) rooted in semantically annotated [JSON-LD](https://json-ld.org/) transactions, leveraging the [RO-Crate](https://www.researchobject.org/ro-crate/) metadata packaging standard. Additional efforts have already begun in and between the [AiiDA](https://www.aiida.net/) workflow manager and [openBIS](https://openbis.ch/) ELN.

**MADICES 2025 Focus Area**
MADICES 3 will extend the applications of the interoperability tools to platforms managing diverse data, from Chemistry and Physics, to life and the materials sciences.

Please include links, comments, and discuss plans for this focus area below.

**Comment by PeterKraus** [comment]
My primary goal for MADICES-2025 is to be able to push data out of my automation software [`tomato`](https://dgbowl.github.io/tomato/main/index.html) into things like datalab, OpenBIS and NOMAD, without having to write a shim for their APIs directly. 

I am looking for a buddy, who:
- could help me understand what the [`interop-demo`](https://github.com/MADICES/interop-demo) from MADICES-2024 is about and how to implement it tomato-side
- is knowledgeable about any ELN (ideally one of the three above) and would be interested in debugging this interoperability during MADICES

**Reply by edan-bainglass** [reply]
I got you @PeterKraus 👍
